scorecard/scorecard/scorecard.py

```diff
@@ -226,57 +233,62 @@ def update_data():
 
     try:
         for repo_name, fq_repo in repos.items():
-            repo = github.get_repo(fq_repo)
-
-            for pr in repo.get_pulls(state='open'):
-                pr_data = get_pr_data(repo, repo_name, pr)
+            async for pr in gh_client.getiter(f'/repos/{fq_repo}/pulls?state=open'):
+                pr_data = await get_pr_data(gh_client, fq_repo, repo_name, pr)
                 new_data[repo_name]['prs'].append(pr_data)
 
-            for issue in repo.get_issues(state='open'):
-                if issue.pull_request is None:
+            async for issue in gh_client.getiter(f'/repos/{fq_repo}/issues?state=open'):
+                print(issue)
+                if 'pull_request' not in issue:
                     issue_data = get_issue_data(repo_name, issue)
                     new_data[repo_name]['issues'].append(issue_data)
-    except RateLimitExceededException as e:
-        log.error('Exceeded rate limit: ' + str(e))
-
+    except Exception:  # pylint: disable=broad-except
+        log.exception('update failed due to except')
```

Comment:
It would be nice to get the actual exception here.

Reply:
Ah nvm, log.exception I think gives the stack trace, so should show it.

Reply:
`log.exception` logs the exception along with the traceback.